Separated AI and User Chat

DodoLintu108 opened the pull request:
This pull request implements the separation of AI responses and user messages into distinct areas within the chat interface. 

### Changes
1. **User Messages Area**:
   - Added a separate section for user messages.
   - Displayed user messages with a distinctive style and avatar.

2. **AI Responses Area**:
   - Added a separate section for AI responses.
   - Displayed AI responses with a different visual style and avatar.

3. **Layout Adjustments**:
   - Ensured a clear separation between user messages and AI responses.
   - Adjusted the layout to improve readability and user experience.

DodoLintu108 committed:
Add detailed description of what you changed

codesmith-emmy reviewed (changes requested):
Can you please change the #header h1 css property from uppercase so that the app name will appear as **gitNarrator** rather than GITNARRATOR where all letters are in uppercase… Also make the font-size around 45-50px.

Can you also make the “User Messages” and “AI Responses” headers static so they remain visible when scrolling through the chat? 

https://github.com/codesmith-emmy/gitNarrator/assets/154705254/fa768db2-7dfd-41e0-8e6d-4ed2b028eea8

Can you also make the **User Messages** area and the **AI Response** area scroll uniformly together so that a user will not easily lose track of the conversation… Also make the **User Message** and **AI Response** be on the same line as illustrated in the screenshot below:

<img width="1080" alt="Screenshot illustration" src="https://github.com/codesmith-emmy/gitNarrator/assets/154705254/b4556558-214e-4426-8f9a-a38e0e5b4ace">


Lastly, please create an avatar for AI in the response area. Thanks

DodoLintu108 committed:
Fixed issue #3: 1. Changed the #header h1 CSS property to display 'gitNarrator' instead of 'GITNARRATOR' by removing the uppercase text-transform property. Increased the font-size to 45-50px.2. Made the 'User Messages' and 'AI Responses' headers static so they remain visible while scrolling through the chat.3. Ensured the 'User Messages' area and 'AI Responses' area scroll uniformly together to maintain conversation context and alignment. 4. Adjusted the layout so that user messages and AI responses are on the same line, enhancing readability and conversational flow.5. Added a distinct avatar for the AI in the response area for better visual distinction.This commit improves the user experience by enhancing readability, maintaining context during scrolling, and providing visual distinctions between user messages and AI responses.

DodoLintu108 commented:
I have made the necessary edits to the files as discussed. The changes include:

- Updated the `#header h1` CSS property to display 'gitNarrator' instead of 'GITNARRATOR' by removing the uppercase text-transform property and adjusting the font size to 45-50px.
- Made the "User Messages" and "AI Responses" headers static to remain visible while scrolling.
- Ensured that the "User Messages" area and the "AI Responses" area scroll uniformly together.
- Adjusted the layout to ensure user messages and AI responses are on the same line for better readability.
- Added a distinct avatar for AI in the response area.

Thank you for reviewing these changes.

DodoLintu108 committed:
Minor Adjustment

codesmith-emmy reviewed (approved):
LGTM
